distributions/LibreELEC/options

```diff
@@ -158,13 +158,13 @@
   INITRAMFS_PARTED_SUPPORT="no"
 
 # build with swap support (yes / no)
-  SWAP_SUPPORT="no"
+  SWAP_SUPPORT="yes"
 
 # swap support enabled per default (yes / no)
   SWAP_ENABLED_DEFAULT="no"
 
 # swapfile size if SWAP_SUPPORT=yes in MB
-  SWAPFILESIZE="256"
+  SWAPFILESIZE="128"
```

Comment:
We have currently everywhere 128mb in use thats why I changed it to the value we currently use. If we keep 256 we would change the current setting - of course this should be a safe value too.